dashboard/modules/job/job_manager.py

```diff
@@ -303,21 +305,15 @@ def _get_driver_env_vars(self, resources_specified: bool) -> Dict[str, str]:
             "PYTHONUNBUFFERED": "1",
         }
 
-    async def _polling(self, child_process) -> int:
-        try:
-            while child_process is not None:
-                return_code = child_process.poll()
-                if return_code is not None:
-                    # subprocess finished with return code
-                    return return_code
-                else:
-                    # still running, yield control, 0.1s by default
-                    await asyncio.sleep(self.SUBPROCESS_POLL_PERIOD_S)
-        except Exception:
```

Comment:
I've removed the except block here. Synced with Jiao and he said there is no specific edge case this is targeting. Also, I am unaware of a case in which poll would throw an exception. If anything really does go wrong, the exception will be caught by the except block in `run()`. Please lmk if anyone has opinions here!